registry/handlers/app.go

```diff
@@ -118,7 +118,7 @@ func NewApp(ctx context.Context, config *configuration.Configuration) *App {
 	if storageParams == nil {
 		storageParams = make(configuration.Parameters)
 	}
-	storageParams["useragent"] = fmt.Sprintf("docker-distribution/%s %s", version.Version, runtime.Version())
+	storageParams["useragent"] = fmt.Sprintf("distribution/%s %s", version.Version, runtime.Version())
```

Comment:
I guess `cncf-distribution` could be an alternative.

That said; it looks like this unconditionally overwrites the option (if present) from `storageParams`, which may be intentional (if it should not be configurable through the config-file), but wondering if this should be passed as argument: if this code is also used as library code in other products, it should probably be configurable?